src/components/registerRunningDocker/registerRunningDocker.tsx

```
@@ -41,20 +42,21 @@ const RegisterRunningDocker: React.FC = () => {
+
   const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
     setNewUrl(e.target.value);
   };
 
   const handleAddUrl = () => {
     if (newUrl.trim() !== '') {
-      const fullUrl =
-        newUrl != 'devnet.spaceshard.io' ? 'http://' + newUrl : 'https://devnet.spaceshard.io';
-      const urlExists = urlList.some((devnet) => devnet.url === fullUrl);
-      if (!urlExists) {
+      const fullUrl = newUrl != 'devnet.spaceshard.io' ? 'http://' + newUrl : DEFAULT_DEVNET_URL;
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)